docs/MarkdownInput.md

```diff
@@ -30,4 +29,165 @@ const PostEdit = () => (
 );
 ```
 
+## Props
+
+`<MarkdownInput>` accepts the following props:
+
+| Prop                 | Required | Type      | Default    | Description                                                           |
+| -------------------- | -------- | --------- | ---------- | --------------------------------------------------------------------- |
+| `source`             | Required | `string`  |            | The field name in the record                                          |
+| `fullWidth`          | Optional | `boolean` | `true`     | If `true`, the input will expand to fill the form width               |
+| `label`              | Optional | `string`  |            | The label. If not set, the label is inferred from the child component |
+| `helperText`         | Optional | `string`  |            | The helper text to display under the input                            |
+| `previewStyle`       | Optional | `string`  | `vertical` | Markdown editor's preview style. Can be `tab` or `vertical`           |
+| `height`             | Optional | `string`  | `512px`    | Markdown editor's height                                              |
+| `initialEditType`    | Optional | `string`  | `wysiwyg`  | Markdown editor's initial edit type. Can be `markdown` or `wysiwyg`   |
+| `useCommandShortcut` | Optional | `boolean` | `true`     | Whether use keyboard shortcuts to perform commands                    |
+
+`<MarkdownInput>` also accepts the [common input props](https://marmelab.com/react-admin/Inputs.html#common-input-props) and the [editor props](https://nhn.github.io/tui.editor/latest/ToastUIEditorCore) from the Toast UI editor.
+
+## `source`
+
+Specifies the field of the record that the input should edit. It is required.
+
+{% raw %}
+
+```jsx
+<Form record={{ id: 123, title: 'Hello, world!', body: '<p>Lorem Ipsum</p>' }}>
+    <MarkdownInput source="body" />
+    {/* default value is "<p>Lorem Ipsum</p>" */}
+</Form>
+```
+
+{% endraw %}
+
+## `fullWidth`
+
+You can make the markdown editor full width by setting the `fullWidth` prop to `true`:
+
+{% raw %}
+
+```jsx
+<SimpleForm>
+    <MarkdownInput source="description" fullwidth />
+</SimpleForm>
+```
+
+{% endraw %}
+
+## `helperText`
+
+If you need to display a text below the markdown editor (usually to explain the expected data to the user), use the `helperText` prop.
+
+{% raw %}
+
+```jsx
+<SimpleForm>
+    <MarkdownInput
+        source="description"
+        helperText="Enter a description of the post"
+    />
+</SimpleForm>
+```
+
+{% endraw %}
+
+## `previewStyle`
+
+You can customize the preview style by setting the `previewStyle` prop. It accepts `tab` or `vertical` and is set to `vertical` by default.
```

Comment:
I'm not sure I understand what this prop does. You either need to explain it differently, or to add screenshots.